src/components/picker/types.tsx

```diff
@@ -48,85 +48,68 @@ export interface PickerSearchStyle {
    */
-  value?: PickerValue;
+  getItemValue?: (value: PickerValue) => any;
   /**
-   * Callback for when picker value change
+   * @deprecated
+   * A function that extract the label out of the value prop in case value has a custom structure (e.g. {myValue, myLabel})
    */
-  onChange?: (value: PickerValue) => void;
+  getItemLabel?: (value: PickerValue) => string;
   /**
-   * SINGLE mode or MULTI mode
+   * @deprecated
+   * Callback for modal onShow event
+   * Instead pass onShow via customPickerProps.modalProp | pickerModalProps
    */
-  mode?: PickerModes | `${PickerModes}`;
+  onShow?: () => void;
   /**
-   * Limit the number of selected items
+   * @deprecated
+   * instead pass items
    */
-  selectionLimit?: number;
+  children?: ReactNode | undefined;
   /**
-   * Adds blur effect to picker modal (iOS only)
+   * @deprecated
+   * Render a custom header for Picker's dialog
+   * instead use renderCustomOverlayHeader
    */
-  enableModalBlur?: boolean;
+  renderCustomDialogHeader?: (callbacks: {onDone?: () => void; onCancel?: () => void}) => React.ReactElement;
   /**
+   * @deprecated
    * Render custom picker - input will be value (see above)
    * Example:
    * renderPicker = (selectedItem) => {...}
+   * instead use renderInput
    */
   renderPicker?: RenderPicker;
   /**
-   * Render custom picker item
-   */
-  renderItem?: (
-    value: PickerValue,
-    itemProps: PickerItemProps & {isSelected: boolean; isItemDisabled: boolean},
-    label?: string
-  ) => React.ReactElement;
-  /**
+   * @deprecated
    * Render custom picker modal (e.g ({visible, children, toggleModal}) => {...})
+   * instead use renderPickerOverlay
    */
   renderCustomModal?: (modalProps: RenderCustomModalProps) => React.ReactElement;
-  /**
-   * Custom picker props (when using renderPicker, will apply on the button wrapper)
-   */
-  customPickerProps?: ExpandableOverlayProps;
-  /**
-   * Add onPress callback for when pressing the picker
-   */
-  onPress?: () => void;
   /**
    * @deprecated
-   * A function that extract the unique value out of the value prop in case value has a custom structure (e.g. {myValue, myLabel})
+   * Pass props to the picker modal
+   * instead pass modalProps via customPickerProps.modalProps
    */
-  getItemValue?: (value: PickerValue) => any;
+  pickerModalProps?: ExpandableOverlayProps['modalProps'];
   /**
    * @deprecated
-   * A function that extract the label out of the value prop in case value has a custom structure (e.g. {myValue, myLabel})
-   */
-  getItemLabel?: (value: PickerValue) => string;
-  /**
-   * A function that returns the label to show for the selected Picker value
-   */
-  getLabel?: (value: PickerValue) => string;
-  /**
-   * The picker modal top bar props
+   * Custom container style
+   * instead use inputContainerStyle
    */
-  topBarProps?: ModalTopBarProps;
+  containerStyle?: StyleProp<ViewStyle>;
```

Comment:
I don't think 'containerStyle' should be renamed. ContainerStyle is a proper name for style passed to the container, which in this case is the input's container.